manifest.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
note that in @sphinxc0re 's implementation of holochain-cmd uses specific names for each file, not a generic name (manifest.json), so this needs to be reconciled on one side or the other.

Reply:
@lucksus and @sphinxc0re and @zippy spent some time talking about this, and the role of the packaging tool.  @lucksus and @zippy came down on the side of making the packager very simple and not really having a deep pre-understanding of the exact format of the directory and it's json contents, but rather simply to be able to take any tree and assemble what ever json files it finds there into a single json blob, with the smarts of also being able to compile the wasm code in finds there and add it into a "code" blob.  That way we can, in this initial dev phase use change the tree structure easily without having to recompile the packager tool.